app/components/tts-config.tsx

```
@@ -88,23 +129,44 @@ export function TTSConfigList(props: {
+                      );
+                }}
+              />
+
+              <Select
+                value={props.ttsConfig.voice}
+                onChange={(e) => {
+                  props.updateConfig(
+                    (config) =>
+                      (config.voice = TTSConfigValidator.voice(
+                        e.currentTarget.value,
+                      )),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)